feature-group-definitions/selectlist.yml

```diff
@@ -0,0 +1,3 @@
+name: "<selectlist>"
```

Comment:
The explainer has now been changed to `<select>` instead of `<selectlist>`: https://open-ui.org/components/selectlist/